jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -1754,67 +1746,6 @@ YUI.add('juju-topology-service', function(Y) {
             flash: { hideHelp: true }
           }
         }});
-      this.showServiceMenu(box);
-    },
-
-    /**
-     * Show the service menu.
-     *
-     * @method showServiceMenu
-     * @param {object} box The presentation state for the service.
-     * @return {undefined} Side effects only.
-     */
-    showServiceMenu: function(box) {
-      var serviceMenu = this.get('container').one('#service-menu');
-      var topo = this.get('component');
-      var service = box.model;
-      var triangle = serviceMenu.one('.triangle');
-
-      if (box && !serviceMenu.hasClass('active')) {
-        topo.set('active_service', box);
-        topo.set('active_context', box.node);
-        serviceMenu.addClass('active');
-
-        // Remove is-selected class from all services and add to the currently
-        // clicked service.
-        if (box.node) {
-          topo.vis.selectAll('.is-selected').classed('is-selected', false);
```

Comment:
@makyo we want to keep this functionality. When clicking on the canvas it should de-select services.